### ISSUE TITLE
MangaNato image scraping blocked by Cloudflare 

### ISSUE THREAD

**Issue by 1Hira0** [1Hira0/Shita#1]
https://github.com/1Hira0/Shita/blob/Work-in-progress/src/backend/m-nato/m-nato.ts#L44

**Comment by 1Hira0** [comment]
Postponed till motivation and time. Puppeteer is here to help
